resources/views/welcome.blade.php

```diff
@@ -194,6 +194,114 @@
     </section>
 
     <x-numbers />
-    <x-contact />
+    <section class="contact-section">
```

Comment:
Remember you do not have to repeat yourself.

Keep the form in one place which in this case it is `<x-contact />` so that the same code can be used on the homepage and contact page